.tekton/gitops-backend-pull-request.yaml

```diff
@@ -26,8 +26,18 @@ spec:
     value: quay.io/redhat-user-workloads/rh-openshift-gitops-tenant/openshift-gitops-operator/gitops-backend:on-pr-{{revision}}
   - name: image-expires-after
     value: 5d
+  - name: hermetic
+    value: "true"
+  - name: prefetch-input
+    value: '{"type": "gomod", "path": "."}'    
+  - name: build-platforms
+    value:
+    - linux/x86_64
+    - linux/arm64
+    - linux/ppc64le
+    - linux/s390x  
```

Comment:
```suggestion
```
Let's run PR builds only on amd64. I noticed long delays in setting up build env for non amd archs.